sci-misc/boinc/boinc-7.6.25.ebuild

```diff
@@ -0,0 +1,141 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
Dear Sven,
I totally overlooked `wxwidgets.eclass`. So yes, stay at `EAPI=5` for the moment, but make it `EAPI=6`-ready, i.e., remove `autotools-utils`, and include the rest of the improvements.